core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -1122,7 +1122,7 @@ public ClusterExecutor executor() {
       // Have to make sure the transport is running before we retrieve it
       Transport transport = globalComponentRegistry.getComponent(BasicComponentRegistry.class).getComponent(Transport.class).running();
       if (transport != null) {
-         long time = getCacheManagerConfiguration().transport().distributedSyncTimeout();
+         long time = configurationManager.getGlobalConfiguration().transport().distributedSyncTimeout();
```

Comment:
Don't forget to change the permission to EXEC, maybe update the EXEC javadoc as well

Reply:
Yes, tbh I totally forgot I even made this. haha